lib/formatPhone.js

```diff
@@ -0,0 +1,20 @@
+/* global formatPhone:true */
+/* exported formatPhone */
+
+formatPhone = phone => {
+  if (!phone) {
+    return null
+  }
+  const phoneDigits = phone.toString().replace(/\D/g, '')
+  const areaCode = phoneDigits.slice(0, 3)
+  const prefix = phoneDigits.slice(3, 6)
+  const suffix = phoneDigits.slice(6, 10)
+  let formatted = String(areaCode)
+  if (phoneDigits.length > 3) {
+    formatted = `(${areaCode}) ${prefix}`
+  }
+  if (phoneDigits.length > 6) {
+    formatted += `-${suffix}`
+  }
+  return formatted
```

Comment:
Actually, the _correct_ solution for this is: https://github.com/LearnersGuild/idm/issues/110

I actually copy-and-pasted this `formatPhoneNumber` code from IDM, which prevents you from completing sign-up unless you've got an area code. Once we switch to E.164 in IDM, we'll need to update this code. So, now that I'm thinking about it, we should format things as E.164 here so that there's no future break.